soc/riscv/riscv-privilege/nuclei/gd32vf103/Kconfig.defconfig.series

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021 Tokita, Hiroshi <tokita.hiroshi@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_RISCV_GIGADEVICE_GD32VF103
+
+source "soc/riscv/riscv-privilege/nuclei/gd32vf103/Kconfig.defconfig.gd32vf103*"
+
+config KERNEL_ENTRY
+	default "__nuclei_start"
+
+config SOC_SERIES
+	default "nuclei"
+
+config NUCLEI_CORE_VARIANT
+	default "gd32vf103"
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 32768
+
+config RISCV_SOC_INTERRUPT_INIT
+	default y
+
+config RISCV_HAS_CPU_IDLE
+	default y
+
+config RISCV_HAS_PLIC
+	default n
+
+config NUM_IRQS
+	default 4096 if NUCLEI_ECLIC
+	default 64  if !NUCLEI_ECLIC
+
+config FLASH_BASE_ADDRESS
+	default $(dt_node_reg_addr_hex,flash0@8000000)
+
+config 2ND_LEVEL_INTERRUPTS
+	default y
+
+config PINCTRL
+	default y
+
+config PINCTRL_GD32_AFIO
+	default y
```

Comment:
please drop this, just make sure SoC selects `GD32_HAS_AFIO_PINMUX`. If pinctrl node is enabled, the driver will be automatically enabled too.

Reply:
I'll apply it.